code/go/0chain.net/blobber/config.go

```diff
@@ -132,6 +131,10 @@ func setupConfig(configDir string, deploymentMode int) {
 	config.Configuration.WebsiteUrl = viper.GetString("website_url")
 	config.Configuration.LogoUrl = viper.GetString("logo_url")
 	config.Configuration.Description = viper.GetString("description")
+	config.Configuration.Version = viper.GetString("version")
```

Comment:
add `version` in yaml file, it will have the same issue with keep_tables. it has a deployment issue. why it is required here for AutoMigrate?